src/api/messages/sendMessage.js

```diff
@@ -9,10 +9,10 @@ export default async (
   params: {|
```

Comment:
Yeah, even I found this strange.

Reply:
`subject` is the old name for `topic`.

Dunno what the story is with `local_id` and `queue_id` -- it looks like those aren't documented, but I'd sure expect that they're still supported, as they're how we match up the eventual `message` / EVENT_NEW_MESSAGE event with the right outbox message to forget about. They may just never have been documented, though I thought our API-docs infrastructure was at the point now where it'd flag that. Would be good to ask in chat.

Reply:
> `subject` is the old name for `topic`.

True, I know, but it would be good to know more; maybe we can start using the new name (maybe deeper changes are needed to allow this?), when was the switch made, etc.

Reply:
Tim responded [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/https.3A.2F.2Fzulip.2Ecom.2Fapi.2Fsend-message/near/915346).